app/components/menubar.js

```diff
@@ -1,11 +1,11 @@
-import React, {useState } from 'react';
-import { Navbar, Nav, NavDropdown, Container, Dropdown } from 'react-bootstrap';
+import Cookies from 'js-cookie';
+import Link from 'next/link';
+import React, { useState } from 'react';
+import { Container, Dropdown, Nav, Navbar, NavDropdown } from 'react-bootstrap';
```

Comment:
Clean this up - you moved the headers around on all files.  Thanks.

Reply:
Cleaned it, happened coz of vs-code automatic imports organise setting